tests/core/a-entity.test.js

```diff
@@ -680,6 +680,19 @@ suite('a-entity', function () {
       el.setAttribute('class', 'pied piper');
       assert.equal(el.getAttribute('class'), 'pied piper');
     });
+
+    test('the attribute cache only stores the modified properties', function () {
+      var el = this.el;
+      el.setAttribute('geometry', {primitive: 'sphere', radius: 10});
```

Comment:
I would assert the `attrValue` before the change too for clarity. Perhaps start with a box.